spring-cloud-gcp-samples/spring-cloud-gcp-security-iap-sample/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
Maybe add the commented out audience property? Otherwise, you can just delete this file.